doc/hardware/peripherals/retained_mem.rst

```diff
@@ -17,6 +17,20 @@ Related configuration options:
 
 * :kconfig:option:`CONFIG_RETAINED_MEM`
 * :kconfig:option:`CONFIG_RETAINED_MEM_INIT_PRIORITY`
+* :kconfig:option:`CONFIG_RETAINED_MEM_MUTEX_FORCE_DISABLE`
+
+Mutex protection
+****************
+
+Mutex protection of retained memory drivers is enabled by default when
+applications are compiled with multithreading support, this means that
```

Comment:
minor: I think there should be `.` after "support", i.e. begin a new sentence after that

Reply:
Fixed